docs/content/en/configuration/users.md

```diff
@@ -15,5 +15,16 @@ Requests to `chproxy` must be authorized with credentials from [user_config](htt
 
 Limits for `in-users` and `out-users` are independent.
 
-`in-users` with `is_wildcarded` flag `true` have names that look like `<prefix>_*` (e.g `analyst_*`).
-Asterisk matches a sequence of valid characters (except underscore) in user name in request to `chproxy`. `chproxy` serves wildcarded users in a normal way, except their credentials from incoming requests are resent to ClickHouse as they are, passwords and names of `out-users` are not used for communications with ClickHouse.
+
+`in-users` with `is_wildcarded` flag `true` can bypass chproxy authentication. In this case, the name plays the role of a pattern and must either look like
+* `<prefix>*` (e.g `analyst_*`)
+* `*<suffix>` (e.g `*-UK`)
+* `*`
+The asterisk matches a sequence of valid characters (except asterisk) in user name in request to `chproxy`. `chproxy` serves wildcarded users in a normal way except their credentials from incoming requests are resent to ClickHouse as they are and chproxy doesn't try to authenticate them. Passwords and names of `out-users` are not used for communications with ClickHouse. For security reasons, the default user (that should be disabled in production clickhouse servers) can't work with the wildcarded feature. So, even if you have an `*` wildcarded user, if someone uses chproxy with the user/pwd "default"/"", the query won't go to clickhouse. If you want to use the default user, you have to create a specific default user.
+
+The current implementation of wildcarded doesn't work with some limits on `out-users` (like the max_concurrent_queries) but works well with all the limits on `in-users`. The limits for `in_users` and `out-users` works as if all the users matching the pattern were the same user.
+
+If the wildcarded users are overlapping, the real users will be attached randomly to one of the wildcarded users. For example, let's say:
```

Comment:
Will it remain consistent for the same user? Meaning if `analyst_john-UK` is routed to `analyst1` out user in first place, will it remain the same for the second query launched by him?

Reply:
Technically, while chproxy is up then yes it will be consistent.
If chproxy is restarted, it should be consistent (but I'm not sure).
If we change something and the admin updates chproxy then the behavior is unknown.  
This is why I decided to put "randomly" and don't dive into details so that chproxy admins are warned that they should be extra careful when the wildcarded users are overlapping